.github/workflows/mutation-testing.yml

```
@@ -0,0 +1,42 @@
+# https://help.github.com/en/categories/automating-your-workflow-with-github-actions
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+name: Mutation Tests
+
+jobs:
+
+  mutation-tests:
+    name: Mutation Tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: shivammathur/setup-php@v2
+        with:
+          php-version: 8.0
+          coverage: xdebug
+          tools: composer
+
+      - name: Get composer cache directory
+        id: composer-cache
+        run: echo "dir=$(composer config cache-files-dir)" >> $GITHUB_OUTPUT
+
+      - name: Cache dependencies
+        uses: actions/cache@v3.3.1
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
```

@Chemaclass 

`composer.lock` is currently not checked in and does not exist, so `actions/cache` can not assemble a cache key based on `composer.lock`.

Perhaps change this to

```suggestion
          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.json') }}
```

or check in `composer.lock` - also helps with reproducible builds.

What do you think?

---

You're right. However, we are using the same line in the other workflows ([code-style](https://github.com/gacela-project/gacela/blob/main/.github/workflows/code-style.yml#L33), for example), and no issues there